packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -1,6 +1,7 @@
 import { StatementSyntax, ValueReference } from 'glimmer-runtime';
 import { AttributeBindingReference, RootReference, applyClassNameBinding } from '../utils/references';
 import EmptyObject from 'ember-metal/empty_object';
+import Dict from 'ember-metal/dictionary';
```

Comment:
See the comments in these two files for a relatively good explanation:

* https://github.com/emberjs/ember.js/blob/master/packages/ember-metal/lib/empty_object.js
* https://github.com/emberjs/ember.js/blob/master/packages/ember-metal/lib/dictionary.js

`ember-metal/dictionary` (when used with `null` parent) creates an `EmptyObject` then adds an item and deletes it to force the runtime into keeping it in dictionary mode. Since we don't need this object to be a dictionary, we can just use `new EmptyObject()` directly.